packages/server/test/integration/cypress_spec.js

```diff
@@ -830,7 +830,8 @@ describe('lib/cypress', () => {
     ]
 
     renamedConfigs.forEach(function (config) {
-      it(`logs error and exits when using an old configuration option: ${config.old}`, function () {
+      // TODO: remove this skip once we have a renamed config option - there are none active now
```

Comment:
not sure I understand the skip here? We should probably create a git issue and explain this in more detail and link it back here

Reply:
Currently we don't have any renamed config option errors that are active. So, we haven't renamed any configurations since 10.x (where there would be a key of `newName` in the `breakingOptions`, so I don't have a real use case to test here. There's already a unit test for this behavior, this was checking integration.